<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MailSendDialog</class>
 <widget class="QWidget" name="MailSendDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1390</width>
    <height>913</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0">
   <item>
    <widget class="QFrame" name="frame2">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QGridLayout" name="gridLayout">
        <item row="6" column="2">
         <widget class="QPlainTextEdit" name="txtParagraph"/>
        </item>
        <item row="1" column="2">
         <widget class="QLabel" name="lblAction">
          <property name="text">
           <string>Greeting Card and Virtual Gift Card - (Physical) Home Delivery                                     v1.0</string>
          </property>
         </widget>
        </item>
        <item row="8" column="2">
         <widget class="QLineEdit" name="txtGiftCardAmount">
          <property name="toolTip">
           <string>Proposal Amount</string>
          </property>
          <property name="inputMethodHints">
           <set>Qt::ImhDigitsOnly</set>
          </property>
         </widget>
        </item>
        <item row="10" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="label_2">
          <property name="toolTip">
           <string>Enter Proposal Description</string>
          </property>
          <property name="text">
           <string>Extra Paragraph (seen on card):</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="buddy">
           <cstring>txtParagraph</cstring>
          </property>
         </widget>
        </item>
        <item row="8" column="0">
         <widget class="QLabel" name="label_212">
          <property name="toolTip">
           <string>Enter Proposal Amount</string>
          </property>
          <property name="text">
           <string>Virtual Gift Card Amount USD:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="buddy">
           <cstring>txtGiftCardAmount</cstring>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_213">
          <property name="toolTip">
           <string>Enter Proposal Name</string>
          </property>
          <property name="text">
           <string>Recipient Name:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="buddy">
           <cstring>txtReserved</cstring>
          </property>
         </widget>
        </item>
        <item row="5" column="2">
         <widget class="QLineEdit" name="txtName">
          <property name="toolTip">
           <string>Address</string>
          </property>
          <property name="inputMethodHints">
           <set>Qt::ImhNone</set>
          </property>
          <property name="echoMode">
           <enum>QLineEdit::Normal</enum>
          </property>
         </widget>
        </item>
        <item row="14" column="0">
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item row="7" column="2">
         <widget class="QLineEdit" name="txtReserved">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="toolTip">
           <string>Enter Proposal Name</string>
          </property>
         </widget>
        </item>
        <item row="9" column="0">
         <widget class="QLabel" name="label_22">
          <property name="toolTip">
           <string>Discussion URL</string>
          </property>
          <property name="text">
           <string>Street Address Line 1:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
          <property name="buddy">
           <cstring>txtAddressLine1</cstring>
          </property>
         </widget>
        </item>
        <item row="9" column="2">
         <widget class="QLineEdit" name="txtAddressLine1">
          <property name="toolTip">
           <string>Discussion URL</string>
          </property>
         </widget>
        </item>
        <item row="10" column="0">
         <widget class="QLabel" name="label194">
          <property name="toolTip">
           <string/>
          </property>
          <property name="text">
           <string>Street Address Line 2:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="10" column="2">
         <widget class="QLineEdit" name="txtAddressLine2">
          <property name="toolTip">
           <string>Address</string>
          </property>
          <property name="inputMethodHints">
           <set>Qt::ImhNone</set>
          </property>
          <property name="echoMode">
           <enum>QLineEdit::Normal</enum>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="11" column="0">
         <widget class="QLabel" name="label1941">
          <property name="toolTip">
           <string/>
          </property>
          <property name="text">
           <string>City:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="11" column="2">
         <widget class="QLineEdit" name="txtCity">
          <property name="toolTip">
           <string>Address</string>
          </property>
          <property name="inputMethodHints">
           <set>Qt::ImhNone</set>
          </property>
          <property name="echoMode">
           <enum>QLineEdit::Normal</enum>
          </property>
          <property name="readOnly">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="12" column="0">
         <widget class="QLabel" name="label1942">
          <property name="toolTip">
           <string/>
          </property>
          <property name="text">
           <string>State:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="12" column="2">
         <widget class="QLineEdit" name="txtState">
          <property name="toolTip">
           <string>Address</string>
          </property>
          <property name="inputMethodHints">
           <set>Qt::ImhNone</set>
          </property>
          <property name="echoMode">
           <enum>QLineEdit::Normal</enum>
          </property>
          <property name="readOnly">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="13" column="0">
         <widget class="QLabel" name="label1945">
          <property name="toolTip">
           <string/>
          </property>
          <property name="text">
           <string>City:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="13" column="2">
         <widget class="QLineEdit" name="txtZip">
          <property name="toolTip">
           <string>Address</string>
          </property>
          <property name="inputMethodHints">
           <set>Qt::ImhNone</set>
          </property>
          <property name="echoMode">
           <enum>QLineEdit::Normal</enum>
          </property>
          <property name="readOnly">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="21" column="0">
         <widget class="QLabel" name="label_31">
          <property name="toolTip">
           <string>Expense Type</string>
          </property>
          <property name="text">
           <string>Card Type:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="21" column="2">
         <widget class="QComboBox" name="cmbType"/>
        </item>
        <item row="23" column="2">
         <widget class="QCheckBox" name="chkDeliver">
          <property name="maximumSize">
           <size>
            <width>777</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Deliver this for real? (If unchecked we will create a proof for review, if checked we will charge you and deliver it).</string>
          </property>
         </widget>
        </item>
        <item row="24" column="2">
         <widget class="QPushButton" name="btnSubmit">
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Send</string>
          </property>
         </widget>
        </item>
        <item row="24" column="7">
         <layout class="QHBoxLayout" name="horizontalLayout"/>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="label_6">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="txtInfo">
          <property name="font">
           <font>
            <pointsize>15</pointsize>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>txtName</tabstop>
  <tabstop>txtParagraph</tabstop>
  <tabstop>txtReserved</tabstop>
  <tabstop>txtGiftCardAmount</tabstop>
  <tabstop>txtAddressLine1</tabstop>
  <tabstop>txtAddressLine2</tabstop>
  <tabstop>txtCity</tabstop>
  <tabstop>txtState</tabstop>
  <tabstop>txtZip</tabstop>
  <tabstop>cmbType</tabstop>
  <tabstop>chkDeliver</tabstop>
  <tabstop>btnSubmit</tabstop>
 </tabstops>
 <resources>
  <include location="../biblepay.qrc"/>
 </resources>
 <connections/>
</ui>
